lib/Robo/Plugin/Commands/TestEnvironmentCommands.php

```diff
@@ -117,27 +117,37 @@ public function configureTests(
     }
 
     /**
-     * Download and run the chrome web driver
+     * Download and run the Chrome WebDriver.
+     * @command driver:run-chrome
      * @param array $opts
+     * @option string $url_base 
+     * @option bool $reinstall Forces the Chrome WebDriver executable to be reinstalled, can be used to get a newer version.
+     * @usage driver:run-chrome --reinstall
      */
-    public function driverRunChrome($opts = ['url_base' => '/wd/hub'])
+    public function driverRunChrome($opts = ['url_base' => '/wd/hub', 'reinstall' => false])
     {
         $this->say('Driver Run Chrome');
         $os = new OperatingSystem();
         $paths = new Paths();
         $url = $this->getChromeWebDriverUrl();
-        $basePath = $os->toOsPath($paths->getProjectPath().'/build/tmp/');
+        $basePath = $os->toOsPath($paths->getProjectPath() . '/build/tmp/');
 
         if (!file_exists($basePath)) {
             if (mkdir($basePath, 0777, true) === false) {
-                throw  new \RuntimeException('Unable to create file structure ' . $basePath);
+                throw new \RuntimeException('Unable to create file structure ' . $basePath);
+            }
+        } else if ($opts['reinstall']) {
+            $this->_deleteDir($basePath);
+            if (mkdir($basePath, 0777, true) === false) {
```

Comment:
```suggestion
            if (mkdir($basePath, 0777, true) || is_dir($basePath)) {
```

Reply:
@Dillon-Brown This caused problems because if `mkdir` worked, the exception would be thrown. It'd also cause `is_dir` to return true, which also causes the exception.